eng/dnx/nuspec/README.md

```diff
@@ -57,7 +57,7 @@ If you'd like to use a specific version of the Azure MCP server, you can specify
 			"--source",
 			"https://api.nuget.org/v3/index.json",
 			"--version",
-			"0.6.0",
+			"0.7.0",
```

Comment:
do we need this in the /end/npx/README.md too?

Reply:
I don't think we ever mention the version in that one. By the way, did you mean the `eng/npm/README.md`?

Reply:
Yes, I meant the eng/npm/README.md..